src/Frontend/src/composables/serviceLicense.ts

```diff
@@ -66,11 +59,14 @@ const licenseStatus = reactive({
   warningLevel: LicenseWarningLevel.None,
 });
 
-async function useLicense() {
-  watch<UnwrapNestedRefs<License>>(license, async (newValue, oldValue) => {
+let router: ReturnType<typeof useRouter>;
+
+async function useLicense(vueRouter: ReturnType<typeof useRouter>) {
+  router = vueRouter;
+  watch<UnwrapNestedRefs<License>>(license, (newValue, oldValue) => {
     const checkForWarnings = oldValue !== null ? newValue && newValue.license_status != oldValue.license_status : newValue !== null;
     if (checkForWarnings) {
-      displayWarningMessage(newValue.license_status);
+      displayWarningMessage(LicenseStatus.ValidWithExpiringUpgradeProtection);
```

Comment:
```suggestion
      displayWarningMessage(newValue.license_status);
```
Removing testing value